README.md

```diff
@@ -1,5 +1,17 @@
-[![Build Status](https://travis-ci.com/heremaps/here-olp-edge-sdk-cpp.svg?branch=master)](https://travis-ci.com/heremaps/here-olp-edge-sdk-cpp)
-[![Build Status](https://dev.azure.com/heremaps/github/_apis/build/status/heremaps.here-olp-edge-sdk-cpp?branchName=master)](https://dev.azure.com/heremaps/github/_build/latest?definitionId=1&branchName=master)
+&nbsp;
+
+| Linux                          | Windows                         |
```

Comment:
I would not put the table before the h1 name anymore.
I would move it if you need a table after "The HERE OLP Edge SDK is a work in progress ... " sentence.
Or remove the table and only leave badges as before until we figure out how to do it. I think we need an bigger table with all supported and verified platforms and then we can add badges there. Leo is preparing something like this.